src/Chainweb/CutDB.hs

```diff
@@ -311,11 +312,24 @@ processCuts logFun headerStore payloadStore queue cutVar = queueToStream
     threshold :: Int
     threshold = int $ 2 * diameter graph * order graph
 
+    -- NOTE: THIS NUMBER MUST BE LARGER THAN THE RESPECTIVE LIMIT
+    -- IN 'CutDB.Sync'
+    farAHeadThreshold :: Int
+    farAHeadThreshold = 2000
```

Comment:
This makes me nervous.

Reply:
This will be fixed in the Rocksdb PR #163